Sources/Application/Environment/Configuration/Debug.xcconfig

```diff
@@ -0,0 +1,16 @@
+INFOPLIST_KEY_CFBundleDisplayName = One Login - Debug
+INFOPLIST_KEY_NSCameraUsageDescription = This app does not use the camera
```

Comment:
We need to pick this one up with content designers: we do now have some camera functionality with QR scanning 🙂

Reply:
Defect raised to address this: DCMAW-10156